packages/serverless/src/apollo/app/document.ts

```diff
@@ -326,12 +278,14 @@ const findDocumentHistory = authorizeWrapper(
 
 const listDocumentHistory = authorizeWrapper<
   TDocumentHistoryListRequest,
-  TDocumentHistoryListResponse
+  TDocumentHistoryResponse[]
```

Comment:
Response should not be an array type, you can alias it

Reply:
fixed